test-vectors/credentials/verify.json

```diff
@@ -6,28 +6,66 @@
       },
-      "errors": true
+      "errors": true,
+      "errorMessage":{
+        "web5-js":"Verification failed: Expected JWT header to contain alg and kid",
+        "web5-kt":"Malformed JWS. Expected header to contain kid."
+     }
     },
     {
       "description": "bad signature",
       "input": {
-        "vcJwt": "eyJraWQiOiJkaWQ6a2V5OnpRM3NoTkx0MWFNV1BiV1JHYThWb2VFYkpvZko3eEplNEZDUHBES3hxMU5aeWdwaXkjelEzc2hOTHQxYU1XUGJXUkdhOFZvZUViSm9mSjd4SmU0RkNQcERLeHExTlp5Z3BpeSIsInR5cCI6IkpXVCJ9.eyJraWQiOiJkaWQ6a2V5OnpRM3NoTkx0MWFNV1BiV1JHYThWb2VFYkpvZko3eEplNEZDUHBES3hxMU5aeWdwaXkjelEzc2hOTHQxYU1XUGJXUkdhOFZvZUViSm9mSjd4SmU0RkNQcERLeHExTlp5Z3BpeSIsInR5cCI6IkpXVCJ9.8AehkiboIK6SZy6LHC9ugy_OcT2VsjluzH4qzsgjfTtq9fEsGyY-cOW_xekNUa2RE2VzlP6FXk0gDn4xf6_r4g"
+        "vcJwt": "eyJ0eXAiOiJKV1QiLCJhbGciOiJFZERTQSIsImtpZCI6ImRpZDpqd2s6ZXlKamNuWWlPaUpGWkRJMU5URTVJaXdpYTNSNUlqb2lUMHRRSWl3aWVDSTZJa0pWTm5KcVlqa3liR015ZVhJMVMwSkJkazE1U2xsUE1qVmlSVWxGTWpCdVltbzFTR1JGZW5RNWRtOGlMQ0pyYVdRaU9pSlJkVmhhVjNCM1J5MDNNMlk1TmtZMlkxcE9TMWQyZG1ablpVTnpla05MWjFFNGNXRkZTR0ZHYUZrNElpd2lZV3huSWpvaVJXUkVVMEVpZlEjMCJ9.eyJ2YyI6eyJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvMjAxOC9jcmVkZW50aWFscy92MSJdLCJ0eXBlIjpbIlZlcmlmaWFibGVDcmVkZW50aWFsIiwiVEJEZXZlbG9wZXJDcmVkZW50aWFsIl0sImlkIjoidXJuOnV1aWQ6YjA0NGFiMzItYzUxNS00ZTgwLWFkNTAtYTIxZWI0NTBlMTU3IiwiaXNzdWVyIjoiZGlkOmp3azpleUpqY25ZaU9pSkZaREkxTlRFNUlpd2lhM1I1SWpvaVQwdFFJaXdpZUNJNklrSlZObkpxWWpreWJHTXllWEkxUzBKQmRrMTVTbGxQTWpWaVJVbEZNakJ1WW1vMVNHUkZlblE1ZG04aUxDSnJhV1FpT2lKUmRWaGFWM0IzUnkwM00yWTVOa1kyWTFwT1MxZDJkbVpuWlVOemVrTkxaMUU0Y1dGRlNHRkdhRms0SWl3aVlXeG5Jam9pUldSRVUwRWlmUSIsImlzc3VhbmNlRGF0ZSI6IjIwMjQtMDMtMDVUMjI6Mzg6MjlaIiwiY3JlZGVudGlhbFN1YmplY3QiOnsiaWQiOiJkaWQ6andrOmV5SmpjbllpT2lKRlpESTFOVEU1SWl3aWEzUjVJam9pVDB0UUlpd2llQ0k2SWtKVk5uSnFZamt5YkdNeWVYSTFTMEpCZGsxNVNsbFBNalZpUlVsRk1qQnVZbW8xU0dSRmVuUTVkbThpTENKcmFXUWlPaUpSZFZoYVYzQjNSeTAzTTJZNU5rWTJZMXBPUzFkMmRtWm5aVU56ZWtOTFoxRTRjV0ZGU0dGR2FGazRJaXdpWVd4bklqb2lSV1JFVTBFaWZRIiwidXNlcm5hbWUiOiJuaXRybyJ9fSwibmJmIjoxNzA5Njc4MzA5LCJqdGkiOiJ1cm46dXVpZDpiMDQ0YWIzMi1jNTE1LTRlODAtYWQ1MC1hMjFlYjQ1MGUxNTciLCJpc3MiOiJkaWQ6andrOmV5SmpjbllpT2lKRlpESTFOVEU1SWl3aWEzUjVJam9pVDB0UUlpd2llQ0k2SWtKVk5uSnFZamt5YkdNeWVYSTFTMEpCZGsxNVNsbFBNalZpUlVsRk1qQnVZbW8xU0dSRmVuUTVkbThpTENKcmFXUWlPaUpSZFZoYVYzQjNSeTAzTTJZNU5rWTJZMXBPUzFkMmRtWm5aVU56ZWtOTFoxRTRjV0ZGU0dGR2FGazRJaXdpWVd4bklqb2lSV1JFVTBFaWZRIiwic3ViIjoiZGlkOmp3azpleUpqY25ZaU9pSkZaREkxTlRFNUlpd2lhM1I1SWpvaVQwdFFJaXdpZUNJNklrSlZObkpxWWpreWJHTXllWEkxUzBKQmRrMTVTbGxQTWpWaVJVbEZNakJ1WW1vMVNHUkZlblE1ZG04aUxDSnJhV1FpT2lKUmRWaGFWM0IzUnkwM00yWTVOa1kyWTFwT1MxZDJkbVpuWlVOemVrTkxaMUU0Y1dGRlNHRkdhRms0SWl3aVlXeG5Jam9pUldSRVUwRWlmUSIsImlhdCI6MTcwOTY3ODMwOX0.badsig"
       },
-      "errors": true
+      "errors": true,
+      "errorMessage":{
+        "web5-js":"signature expected 64 bytes, got 4",
```

Comment:
+1, though personally i take it a step further: I would prefer not to check error messages even if they're the same per SDK. Different languages surface errors differently, and standardizing error messages feels like standardizing an implementation detail.

Reply:
yes, that's the decision i'm coming to as well @diehuxx . checking error messages is rife with tests failing for the worst reasons - a typo.

Reply:
specifying error message can be done, but usually at the API layer
since the spec does not currently specify specific error modes it does feel out of place to add these in here

so agree that - we should be testing the presence of an error but not the contents of its message

Reply:
Sounds good,

I will remove these and I added a issue to get more grainular error messages. 
https://github.com/TBD54566975/web5-spec/issues/147